application/Common/Shared/Utility/NativeMethods.cs

```diff
@@ -76,6 +76,38 @@ public static bool IsKeyPressed(VirtualKeyCode virtualKeyCode)
 
         #endregion
 
+        #region Clipboard text helper
+
+        /// <summary>
+        ///     Gets the text from the clipboard
+        /// </summary>
+        /// <param name="hwnd">Pointer to the window that currently has clipboard</param>
+        /// <returns>String representing text from the clipboard</returns>
+        public static string getClipboardText()
+        {
+            uint CF_TEXT = 1;
+            uint CF_UNICODETEXT = 13;
+
+            OpenClipboard(GetOpenClipboardWindow());
+
+            //Gets pointer to clipboard data in the selected format
+            var ClipboardDataPointer = GetClipboardData(CF_UNICODETEXT);
+
+            var gLock = GlobalLock(ClipboardDataPointer);
```

Comment:
`gLock` is quite unspecific. I would suggest `clipboardLock` or similar instead.